README.md

```diff
@@ -1,41 +1,12 @@
-# Crate Name
+# vhost-device
 
 ## Design
 
-TODO: This section should have a high-level design of the crate.
-
-Some questions that might help in writing this section:
-- What is the purpose of this crate?
-- What are the main components of the crate? How do they interact which each
-  other?
+This repository hosts various 'vhost-user' device backends in their own crates.
+See their individual README.md files for specific information about those
```

Comment:
How about also mentioning all these devices (which right now is just one :laughing:) here as well, and have a link that takes you to their readme?

Reply:
done